### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by chrisglein**
Update templates to be more succinct so it is more likely people use them.

- All templates contain a comment link to our triage process, so there's some transparency about what happens next (in the future we can include SLAs there for questions/bugs/features
- The issue template is now almost identical to the core repo one, and substantially less verbose with stuff you need to delete
- Removed RNW from the titles of the various templates

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/5556)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+<!--
+Your issue will be triaged by the RNW team according to this process: https://github.com/microsoft/react-native-windows/wiki/Triage-Process
+-->
+## Environment
+Run the following in your terminal and copy the results here.
+1. `react-native -v`:
+2. `react-native run-windows --info`:
+3. `reg query "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock"`
```
Comment (chrisglein) [inline .github/ISSUE_TEMPLATE/bug-report.md:17]: Can we gather this with our `info` command instead? Why does this need to be a separate step. In the RN repo you just do `react-native info` and you're done, where we have 3 steps.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+<!-- Consider including this information as well:
+What SDK version are you building for? Choose from 10.0.15063, 10.0.16299, 10.0.18362, etc.
+- Target Platform Version(s):
+What device(s) are you targeting? Choose any from Desktop, Xbox, Hololens)
+- Target Device(s):
+Which version of Visual Studio are you using? Specify Visual Studio 2017 or Visual Studio 2019
+- Visual Studio Version:
+Which build configuration are you running? Choose from Debug, DebugBundle, Release, ReleaseBundle 
+- Build Configuration:
+-->
```
Comment (chrisglein) [inline .github/ISSUE_TEMPLATE/bug-report.md:28]: I took this out of the default shown text. How much can we incorporate into `info` and leave out entirely?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+1.
+2.
+
+## Expected Results
+Describe what you expected to happen.
+
+<!--
+Troubleshooting
+  If you see build failure on `react-native run-windows`, please try again with 'react-native run-windows --logging' and provide the output.
```
Comment (chrisglein) [inline .github/ISSUE_TEMPLATE/bug-report.md:41]: This was an odd callout. Should we have more troubleshooting items?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+-->
+
+## Snack, code example, screenshot, or link to a repository:
+Please provide a Snack (https://snack.expo.io/), a link to a repository on GitHub, or provide a minimal code example that reproduces the problem.
```
Comment (chrisglein) [inline .github/ISSUE_TEMPLATE/bug-report.md:45]: Kind of unfair given you can't create a Windows Snack. But the old template also referenced it without adequately explaining. Right now this is verbatim from the facebook template. Could add more verbiage to say when to create a Snack.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (asklar) [reply .github/ISSUE_TEMPLATE/bug-report.md:17]: @chrisglein  I looked into this today. The way it works is that there are 2 parts: react-native-community CLI, and envinfo. Envinfo only knowns about android and ios SDKs, editors, etc. and the community CLI only asks for these:
```js
  const info = await _envinfo().default.run({
    System: ['OS', 'CPU', 'Memory', 'Shell'],
    Binaries: ['Node', 'Yarn', 'npm', 'Watchman'],
    IDEs: ['Xcode', 'Android Studio'],
    Managers: ['CocoaPods'],
    Languages: ['Java', 'Python'],
    SDKs: ['iOS SDK', 'Android SDK'],
    npmPackages: ['react', 'react-native', '@react-native-community/cli'],
    npmGlobalPackages: ['*react-native*']
  }, options);
```

We would probably have to add the logic to envinfo to collect Windows-specific things and once that's in place, we can then add it to react-native info

Comment (asklar) [reply .github/ISSUE_TEMPLATE/bug-report.md:17]: @chrisglein there is someone who was going to implement this in envinfo, so I reached out to see if they need help
https://github.com/tabrindle/envinfo/issues/155#issuecomment-661303425

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 3 above)
Comment (asklar) [reply .github/ISSUE_TEMPLATE/bug-report.md:41]: @chrisglein  `--logging` is a good first step. The next step if running into build issues is to open the generated .binlog file in the MSBuild Structured Log Viewer and navigate through the failure to glean some insight into what went wrong

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (chrisglein) [reply .github/ISSUE_TEMPLATE/bug-report.md:28]: SDKS installed is already represented with 'react-native run-windows --info':
```
  System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
    Memory: 5.62 GB / 15.92 GB
  Binaries:
    Node: 12.10.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.17.3 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.10.3 - C:\Program Files\nodejs\npm.CMD
  npmPackages:
    react: 16.11.0 => 16.11.0
    react-native: ^0.62.0-0 => 0.62.2
    react-native-windows: ^0.62.0-0 => 0.62.2
  Installed UWP SDKs:
    10.0.14393.0
    10.0.17763.0
    10.0.18362.0
```
Note that seems to be the only information not also reported with just `react-native info` (other than the RNW version, which is a pretty important bit):
```
info Fetching system and libraries information...
System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
    Memory: 5.71 GB / 15.92 GB
  Binaries:
    Node: 12.10.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.17.3 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.10.3 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
  IDEs:
    Android Studio: Not Found
  Languages:
    Java: Not Found
    Python: 2.7.16
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.11.0 => 16.11.0
    react-native: ^0.62.0-0 => 0.62.2
  npmGlobalPackages:
    *react-native*: Not Found
```
Only additional thing we're getting with `react-native -v` is the CLI version:
```
react-native-cli: 2.0.1
react-native: 0.62.2
```

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (chrisglein) [reply .github/ISSUE_TEMPLATE/bug-report.md:17]: Thanks for following up and filling out that info @asklar with the [PR](https://github.com/tabrindle/envinfo/pull/178). What's the timeline for that roundtripping to affect our setup steps? I assume we take these edits and then refine again when that goes through.

Comment (asklar) [reply .github/ISSUE_TEMPLATE/bug-report.md:17]: I expect the envinfo change to go in soon. After that it's a question of how soon will the CLI accept a change, and how soon after that will they cut a new version; I'm not sure about either of those but probably not immediately.

Comment (chrisglein) [reply .github/ISSUE_TEMPLATE/bug-report.md:17]: I think these changes still have value even while we wait for envinfo (at which point we'll want to reduce the steps even further). Agree?

Comment (asklar) [reply .github/ISSUE_TEMPLATE/bug-report.md:17]: agree

CLI PR here: [Add Windows info to react-native info](https://github.com/react-native-community/cli/pull/1227)

**Pull request merged**